source/_integrations/rachio.markdown

```diff
@@ -49,6 +49,10 @@ After setting up the integration, change the options to set the duration in minu
 
 </div>
 
+### Smart Hose Timers
+
+The Rachio smart hose timers are not currently capable of receiving real-time updates, and instead rely on polling. Because of this, the current state of valves started from a schedule or the physical button will not show up immediately. Polling occurs every 2 minutes when one base station is used, with an additional minute added for every additional base station to remain with the API rate limit. Up to 4 valves can be paired to a single base station.
```

Comment:
```suggestion
The Rachio smart hose timers are not currently capable of receiving real-time updates. Instead, they rely on polling. Because of this, the current state of valves started from a schedule or the physical button will not show up immediately. Polling occurs every 2 minutes when one base station is used, with an additional minute added for every additional base station to remain with the API rate limit. Up to 4 valves can be paired to a single base station.
```